### PR TITLE
Packages all build

### PR DESCRIPTION
**Pull request by maseabunikie**
In this pull request it is possible to generate all debian packages from submodules using packaging/package_all.sh script.

### CODE REVIEW

File: packaging/systemd/package.sh [hunk 1]
```diff
@@ -2,27 +2,23 @@
 
 get_version() {
     pushd ../../systemd
-    version=1.0.0~$(git describe --always --tags --dirty --match "[0-9]*.[0-9]*.[0-9]*")
+    version=1.0.0-$(git log --date=format:%Y%m%d --pretty=dirty~git%cd.%h -n 1)
```
Comment (jnippula) [inline packaging/systemd/package.sh:5]: Does this cause any issue if the first character after the dash is not an incrementing number indicating debian_revision?

Comment (maseabunikie) [reply packaging/systemd/package.sh:5]: The debian package is built. I have not tested yet if it is possible to be installed.

Comment (maseabunikie) [reply packaging/systemd/package.sh:5]: If it is a problem I will replace it with the following format:
version=1.0.0-**0**$(git log --date=format:%Y%m%d --pretty=~git%cd.%h**-dirty** -n 1)

Comment (maseabunikie) [reply packaging/systemd/package.sh:5]: Updated.

**Review by jnippula — APPROVED** [review]

**Pull request merged**